app/views/health/health/goal/hpcs/_hpc.haml

```diff
@@ -91,7 +91,7 @@
             Add
     .row
       .col-sm-6
-        = f.input :responsible_team_member_id, collection: @patient.available_team_members, label: 'Responsible Care Team Member', as: :select_two
+        = f.input :responsible_team_member_id, collection: @patient.available_team_members, label: 'Responsible Care Team Member'
```

Comment:
why isn't this a select 2 anymore?

Reply:
Oh yeah. Good question. These are unusable as select2,  at least for me, because the dropdown gets get off at the modal and you can't scroll it without it disappearing. The only way I could find to fix that is to change the overflow behavior of the modal such that the modal doesn't scroll anymore, which breaks it for smaller screens. On one of the forms (DME or Service, I forget) we were using a non-select2-selector, so I went with that fix instead.

Reply:
That's fine for now, thanks for the details.